android/app/build.gradle

```diff
@@ -35,22 +36,23 @@ configurations.all {
  **/
 
 dependencies {
-    implementation(group: 'com.graphhopper', name: 'graphhopper-core', version: '0.11-SNAPSHOT') {
+    implementation(group: 'com.graphhopper', name: 'graphhopper-core', version: '0.12-SNAPSHOT') {
        exclude group: 'com.google.protobuf', module: 'protobuf-java'
        exclude group: 'org.openstreetmap.osmosis', module: 'osmosis-osm-binary'
        exclude group: 'org.apache.xmlgraphics', module: 'xmlgraphics-commons'
+       exclude group: 'org.locationtech.jts', module: 'jts-core'
     }
 
-    implementation 'org.mapsforge:vtm:0.9.2'
-    implementation 'org.mapsforge:vtm-android:0.9.2'
-    implementation 'org.mapsforge:vtm-android:0.9.2:natives-armeabi'
-    implementation 'org.mapsforge:vtm-android:0.9.2:natives-armeabi-v7a'
-    implementation 'org.mapsforge:vtm-android:0.9.2:natives-arm64-v8a'
-    implementation 'org.mapsforge:vtm-android:0.9.2:natives-x86'
-    implementation 'org.mapsforge:vtm-android:0.9.2:natives-x86_64'
-    implementation 'org.mapsforge:vtm-jts:0.9.2'
-    implementation 'org.mapsforge:vtm-themes:0.9.2'
-    implementation 'com.caverock:androidsvg:1.2.2-beta-1'
+    implementation 'org.mapsforge:vtm:0.10.0'
+    implementation 'org.mapsforge:vtm-android:0.10.0'
+    implementation 'org.mapsforge:vtm-android:0.10.0:natives-armeabi-v7a'
+    implementation 'org.mapsforge:vtm-android:0.10.0:natives-arm64-v8a'
+    implementation 'org.mapsforge:vtm-android:0.10.0:natives-x86'
+    implementation 'org.mapsforge:vtm-android:0.10.0:natives-x86_64'
+    implementation 'org.mapsforge:vtm-jts:0.10.0'
+    implementation 'org.mapsforge:vtm-themes:0.10.0'
+    implementation 'com.caverock:androidsvg:1.3'
+    implementation 'org.locationtech.jts:jts-core:1.15.1'
```

Comment:
Is this required if GH core contains the dependency to jts-core?

Reply:
Ok, will probably just manually add a commit with this dependency change and then we can wait until first snapshot is released.
@michaz are there any objections from your side to downgrade from 1.16 to 1.15.1?

Reply:
@michaz are there any objections from your side to downgrade JTS from 1.16 to 1.15.1?

Reply:
@karussell Not that I know of. I think everything I use in JTS has been there forever.

Reply:
@devemux86 have now made the change and the snapshot is deployed

Reply:
Then can remove the 2 lines:
`exclude group: 'org.locationtech.jts', module: 'jts-core'`
...
`implementation 'org.locationtech.jts:jts-core:1.15.1'`

Reply:
@karussell now should be ok.